src/Workspaces/Core/MSBuild/MSBuild/Build/ProjectBuildManager.cs

```diff
@@ -87,7 +87,10 @@ public ProjectBuildManager(ImmutableDictionary<string, string> additionalGlobalP
 
                 using (var stream = FileUtilities.OpenAsyncRead(path))
                 using (var readStream = await SerializableBytes.CreateReadableStreamAsync(stream, cancellationToken).ConfigureAwait(false))
+#pragma warning disable CA3075 // Insecure DTD processing in XML
+                // s_xmlReaderSettings is compliant with CA3075 (no resolver, prohibit DTD) but the analyzer doesn't pick it up correctly
                 using (var xmlReader = XmlReader.Create(readStream, s_xmlReaderSettings))
```

Comment:
❓ Is there a bug tracking correcting this case? If so, what is the ETA?

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/2105